r/R/csv.R

```diff
@@ -467,6 +473,7 @@ CsvReadOptions <- R6Class("CsvReadOptions",
 CsvReadOptions$create <- function(use_threads = option_use_threads(),
                                   block_size = 1048576L,
                                   skip_rows = 0L,
+                                  skip_rows_after_names = 0L,
```

Comment:
Might it be better to add new parameters at the end of the list, so that this won't break code of users who are already using `CsvReadOptions$create()` and passing in parameters without names?